src/main/webapp/coordinator/degreeSite/viewDescriptionCurricularPlan.jsp

```diff
@@ -40,14 +42,11 @@
 		<html:hidden bundle="HTMLALT_RESOURCES" altKey="hidden.page" property="page" value="1" />
         <bean:define id="degreeCurricularPlanID" name="degreeCurricularPlanID" scope="request" />
 		<html:hidden bundle="HTMLALT_RESOURCES" altKey="hidden.degreeCurricularPlanID" property="degreeCurricularPlanID" value="<%=  degreeCurricularPlanID.toString() %>" />	
-		
-		<fr:edit name="degreeCurricularPlan" type="org.fenixedu.academic.domain.DegreeCurricularPlan" schema="degree.curricular.plan.description" nested="true">
-		    <fr:layout>
-	    	    <fr:property name="classes" value="tstyle2 thlight thright thtop mbottom2"/>
-	        	<fr:property name="columnClasses" value="width10em,pbottom1,tdclear tderror1 valigntop"/>
-		    </fr:layout>
-		</fr:edit>
-	
+
+		<textarea bennu-html-editor bennu-localized-string name="degreeCurricularPlanDescription" id="degreeCurricularPlanDescription">
+			${degreeCurricularPlanDescription.json()}
```

Comment:
Due to encoding issues, and the fact that inside the JSON can be valid HTML, you must escape the JSON when showing it in the textarea, similar to what is done [here](https://github.com/FenixEdu/fenixedu-cms/blob/v1.0.7/src/main/webapp/WEB-INF/fenixedu-cms/editPost.jsp#L86-L87).